src/content/test/semantics/table-semantics.tsx

```diff
@@ -5,23 +5,31 @@ import { create, React } from '../../common';
 export const infoAndExamples = create(({ Markup, Link }) => (
     <>
         <h1>Data tables</h1>
-        <p>Semantic elements in a data table must not be coded as presentational.</p>
+        <p>
+            A <Markup.Code>{'<table>'}</Markup.Code> element must be coded correctly as a data table or layout table.
+        </p>
 
         <h2>Why it matters</h2>
         <p>
-            The semantic elements in a data table communicate relationships between cells and row or column headers. When{' '}
-            <Markup.Code>role="presentation"</Markup.Code> is applied to a data table element, its semantics are suppressed, and critical
-            relationships are consequently hidden from people who use assistive technologies. Moreover, they can't use the standard
-            navigational shortcuts for data tables.
+            The semantic elements in a data table communicate relationships between cells and row or column headers. Applying{' '}
+            <Markup.Code>role="presentation"</Markup.Code> or <Markup.Code>role="none"</Markup.Code> suppresses those semantics.
+        </p>
+        <p>
+            When the semantics in a data table are suppressed, critical relationships are hidden from people who use assistive technologies.
+            Moreover, they can't use the standard navigational shortcuts for data tables.
         </p>
         <p>
-            (Applying <Markup.Code>role="presentation"</Markup.Code> is appropriate for layout tables, because their table semantics do not
-            communicate any meaningful relationships.)
+            When the semantics in a layout table are not suppressed, assistive technologies communicate relationships that don't actually
+            exist.
         </p>
 
         <h2>How to fix</h2>
         <p>
```

Comment:
this two sentences: " Do use role="presentation" on layout tables."  and 
"Do not use role="presentation" or role="none" " should be list elements instead of `<p>`

Reply:
I couldn't tell for sure in the doc if those were list items or not. But it is now fixed.